meta-lxatac-bsp/recipes-core/rauc/rauc-conf.bbappend

```diff
@@ -0,0 +1,52 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+DEPENDS:append = "openssl-native"
+RDEPENDS:${PN}:append = "bash"
```

Comment:
As long as no overrides are used, I'd use simple `+=` for `DEPENDS` and `RDEPENDS` since they are easier to parse and this is consistent with other recipes. However, for the scope of this PR I'd leave it as it was before the split-up, anyway. (Same for `SRC_URI` below.)